src/tests/routes/test_sites.py

```diff
@@ -97,9 +97,9 @@ async def test_fetch_sites(test_app_asyncio, init_test_db):
          201, None],
         [1, {"name": "my_site", "group_id": 1, "lat": 0., "lon": 0.,
              "country": "FR", "geocode": "01"}, True, 201, None],
-        [0, {"name": "my_site", "group_id": 1, "lat": 0., "lon": 0.,
+        [0, {"name": "my_site", "group_id": 2, "lat": 0., "lon": 0.,
              "country": "FR", "geocode": "01"}, False,
-         401, "Permission denied"],
+         401, "You can't specify another group"],
```

Comment:
Sorry, I'm not sure I understand the logic: did the test case change?

If possible, I'd rather keep the group_id 1 and change the expected outcome if it did, and then adding a test case for group_id 2

Reply:
Done